LICENSE

```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright © 2018 Daniel Ng <dan@ngenator.com>
+Copyright © 2020 Nick Silverman <nckslvrmn@gmail.com>
```

Comment:
![yoink](https://user-images.githubusercontent.com/42610668/72553439-d6d00c80-3866-11ea-98d3-6e81c73af5b9.gif)

Reply:
Apparently this is the correct way to alter the MIT license with a new author